src/main/scala/com/horizon/exchangeapi/auth/IbmCloudModule.scala

```diff
@@ -486,7 +490,7 @@ object IbmCloudAuth {
         //logger.error(s"IAM authentication succeeded, but no matching exchange org with a cloud account id was found for ${authInfo.org}")
         DBIO.failed(OrgNotFound(authInfo.org))
       } else if (authInfo.keyType == "iamtoken" && userInfo.accountId == "") {
-        //todo: this is the case with tokens from the edge mgmt ui (because we don't get the accountId when we validate an iamtoken). The ui already verified the org and is using the right one,
+        //todx: this is the case with tokens from the edge mgmt ui (because we don't get the accountId when we validate an iamtoken). The ui already verified the org and is using the right one,
```

Comment:
Minor typo

Reply:
oh, ya, i actually did that on purpose, because that code is commented out. I'm going to get rid of that block of commented out code in the next PR. (I was just really unsure of that change.)